ethcore/types/src/transaction/transaction.rs

```diff
@@ -31,7 +31,8 @@ use transaction::error;
 type Bytes = Vec<u8>;
 type BlockNumber = u64;
 
-/// Fake address for unsigned transactions as defined by EIP-86.
+/// Fake address for unsigned transactions as defined by Legacy EIP-86.
+#[cfg(any(test, feature = "test-helpers"))]
 pub const UNSIGNED_SENDER: Address = H160([0xff; 20]);
```

Comment:
Why not remove this entirely?

Reply:
What to do with
1. https://github.com/paritytech/parity-ethereum/blob/6e76be7fad1cf1ae40773183084f13c37acabdde/ethcore/machine/src/machine.rs#L155-L156
2. https://github.com/paritytech/parity-ethereum/blob/6e76be7fad1cf1ae40773183084f13c37acabdde/ethcore/machine/src/externalities.rs#L262
3. (probably just remove) https://github.com/paritytech/parity-ethereum/blob/6e76be7fad1cf1ae40773183084f13c37acabdde/ethcore/types/src/transaction/transaction.rs#L503-L505

Reply:
1. Try to remove `Option<>` from `contract_address` - I suppose it will always be set to `Some`.
2. Remove along with `keep_unsigned_nonce` parameter from schedule, it's always set to `false` anyway.
3. Just remove along with `fn is_unsigned` methods from all `*Transaction` structs.

Reply:
Just checked and it seems 1. needs to stay, as it's being used by `block-reward` calls inside the engine, so let's leave it as is. So I guess we can keep the constant, but try to remove everything else.